net/developer-guide/presentation-content/manage-ole/_index.md

```diff
@@ -5,65 +5,175 @@ weight: 232
+1. Access OLE Object Frame shape (in this example, we have used the PPTX created above which has only one shape at first slide) and typecast that object as an OLE Object Frame. This was the desired OLE Object Frame to be accessed.
+1. Once OLE Object Frame is accessed, you can perform any operation on it.
 
 In the example given below, an OLE Object Frame (that is a Microsoft Excel Chart object embedded in a slide) is accessed and then all of its Object Data is written to an Excel file.
 
-{{< gist "aspose-slides" "53249e5573d2cd6e66f91f708e8fe008" "Examples-CSharp-Shapes-AccessOLEObjectFrame-AccessOLEObjectFrame.cs" >}}
-## **Change OLE Object data**
-If an OLE object is already embedded in a slide, you can access that object easily using Aspose.Slides for .NET and can modify its data . Please follow the steps below to find how to modify an OLE object data from a slide:
-
-1. Open the desired presentation with embedded Ole Object by creating an instance of [Presentation](http://www.aspose.com/api/net/slides/aspose.slides/presentation) class.
+``` csharp 
+// The path to the documents directory.
+string dataDir = RunExamples.GetDataDir_Shapes();
+
+// Load the PPTX to Presentation object
+using (Presentation pres = new Presentation(dataDir + "AccessingOLEObjectFrame.pptx"))
+{
+    // Access the first slide
+    ISlide sld = pres.Slides[0];
+
+    // Cast the shape to OleObjectFrame
+    OleObjectFrame oleObjectFrame = sld.Shapes[0] as OleObjectFrame;
+
+    // Read the OLE Object and write it to disk
+    if (oleObjectFrame != null)
+    {
+        // Get embedded file data
+        byte[] data = oleObjectFrame.EmbeddedFileData;
+
+        // Get embedded file extention
+        string fileExtention = oleObjectFrame.EmbeddedFileExtension;
+
+        // Create path for saving the extracted file
+        string extractedPath = dataDir + "excelFromOLE_out" + fileExtention;
+
+        // Save extracted data
+        using (FileStream fstr = new FileStream(extractedPath, FileMode.Create, FileAccess.Write))
+        {
+            fstr.Write(data, 0, data.Length);
+        }
+    }
+}
+```
+
+## **Change OLE Object Data**
+If an OLE object is already embedded in a slide, you can access that object easily using Aspose.Slides for .NET and can 
+modify its data. Please follow the steps below to find how to modify an OLE object data from a slide:
+
+1. Open the desired presentation with embedded OLE Object by creating an instance of [Presentation](http://www.aspose.com/api/net/slides/aspose.slides/presentation) class.
 1. Obtain the reference of a slide by using its Index.
 1. Access the OLE Object Frame shape (in this example, we have used the PPTX created above which has only one shape at first slide) and typecast that object as an OLE Object Frame. This was the desired OLE Object Frame to be accessed.
 1. Once the OLE Object Frame is accessed, you can perform any operation on it.
-1. Create the Workbook object and access the Ole Data.

```

Comment:
@optionis 
Write _OLE_ instead _Ole_.
Please, use abbreviations with a capital letter throughout all the article.